api/controllers/UserController.js

```diff
@@ -310,13 +257,12 @@ module.exports = {
     });
   },
 
-  bannedUsers: function (req, res) {
-    User.find({banned: true}).exec(function (err, users) {
-      if (err) {
-        return res.serverError(err);
-      }
-      return res.ok(users);
-    });
+  bannedUsers: async function (req, res) {
+    try {
+      return res.ok(await Users.getBannedUsers());
+    } catch (err) {
+      return res.serverError(err);
+    }
```

Comment:
This is no longer used by the client, but it's nice to have as an api feature. Anyway, it just calls a service now.